english/marshmallow.md

```diff
@@ -350,11 +352,11 @@ Do you:
 [The inception protocol](dream_in_dream/inception.md)
 
 [Choose the blue pill or the red pill?](choose-pill/choose-pill.md)
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.